### ISSUE TITLE
Add python3

### ISSUE THREAD

**Issue by nickoe** [KiCad/kicad-mac-builder#205]
I propose to add python3 such that we can test and verify pointhi's python3 branch on macos. I don't have macos for the time being.

**Comment by nickoe** [comment]
For not it requires to disable wxpython and enabled a new build switch which is documented on his branch.

**Comment by adamwolf** [comment]
I am definitely in favor of this.  I don't think I can start on this until
next Friday.

**Comment by nickoe** [comment]
Although I will be away for this weekend, please keep me updated if you make any progress. :)

**Comment by nickoe** [comment]
@adamwolf Did you get a chance to look into this?

**Comment by adamwolf** [comment]
Yeah, I was a little side tracked with a build breakage, but it's on my
list. I want to get it started this week before any 5.0.1 stuff gets ready.
